certified-connectors/Contacts Pro/Readme.md

```diff
@@ -0,0 +1,41 @@
+## TeamsPro - Contacts Pro Connector
```

Comment:
Readme file doesn't have all the required sections. Please review this file and add required sections. You can refer to this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md).